src/js/events/ServicePlanActions.js

```diff
@@ -0,0 +1,76 @@
+import {
+  REQUEST_PLAN_FETCH_SUCCESS,
+  REQUEST_PLAN_FETCH_ERROR,
+  REQUEST_PLAN_DECISION_SUCCESS,
+  REQUEST_PLAN_DECISION_ERROR
+} from '../constants/ActionTypes';
+
+import AppDispatcher from './AppDispatcher';
+import Config from '../config/Config';
+import RequestUtil from '../utils/RequestUtil';
+
+const ServicePlanActions = {
+
+  fetchPlan: function (serviceID) {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/service${serviceID}${Config.servicePlanAPIPath}`,
```

Comment:
Good catch John. There will always be a prefix of `/` for services. The above works but only because Marathon is smart enough to notice we're missing and encoded `/` at the beginning of the serviceID so it adds it. The correct way of making this request is to encode the serviceID so it includes the `/` prefix as `%2F`, since all serviceID's essentially represent a path in a filesystem.